app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :posts
-  has_many :comments
-  has_many :likes
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
```

Comment:
- [ ] [ OPTIONAL] It would be better to add a default value for `posts_counter, comments_counter, and every_counter ` that way when you try to create a post or user , it will go successfully without rolling back due to validation of posts_counter and comment_counter not a number